osfoffline/tasks/operations.py

```diff
@@ -2,11 +2,24 @@
 import abc
 import asyncio
 
+from sqlalchemy.exc import SQLAlchemyError
 from osfoffline.database_manager import models
 from osfoffline.database_manager.db import session
 from osfoffline.database_manager.utils import save
+from osfoffline.utils.path import ProperPath
+from osfoffline.exceptions.item_exceptions import ItemNotInDB
 
 
+def _get_item_by_path(path):
+    for node in session.query(models.Node):
```

Comment:
Going forward, we'd like to ensure that our code has clear, readable docstrings.

Can you add a docstring to this function? (and include the expected datatype for `path` and for the return value)